catalog/systemd.pl.catalog.in

```diff
@@ -388,3 +388,17 @@ Możliwe są następujące „etykiety”:
   użytkowników (przy wykorzystaniu przestrzeni nazw użytkowników lub NFS)
   nie wynosi 65534.
 Obecny system ma etykietę „@TAINT@”.
+
+-- fe6faa94e7774663a0da52717891d8ef
+Subject: Proces jednostki @UNIT@ został zakończony przez OOM Killer
+Defined-By: systemd
+Support: %SUPPORT_URL%
+
+Proces jednostki @UNIT został zakończony przez mechanizm kończenia
+procesów przy braku pamięci (OOM) jądra Linux. Zwykle wskazuje to,
+że system ma mało pamięci i należało ją zwolnić. Proces powiązany
+z jednostką @UNIT@ został uznany za najlepszy do zakończenia
+i jądro wymusiło zakończenie jego działania.
+
+Proszę zauważyć, że brak pamięci mógł zostać spowodowany
+przez jednostkę @UNIT@.
```

Comment:
mógł zostać spowodowany **nie** przez jednostkę ...

Reply:
The original is `Note that the memory pressure might or might not have been caused by @UNIT@.`, so how about something like `Proszę zauważyć, że brak pamięci mógł, ale nie musiał, zostać spowodowany przez jednostkę @UNIT@.`?

Reply:
Either version works.

Reply:
Even  "mógł nie zostać spowodowany przez jednostkę ...".

Reply:
Let’s go with that last one.